core-java-modules/core-java-arrays-convert/src/main/java/com/baeldung/array/conversions/IntArrayToIterableConversionUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.array.conversions;
+
+
+import com.google.common.primitives.Ints;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class IntArrayToIterableConversionUtils {
+
+    public static Iterable<Integer> convertWithStreamToList(int[] array) {
+        return Arrays.stream(array)
+                .boxed()
```

Comment:
Line continuations must be indented two spaces.

Reply:
<img width="729" alt="image" src="https://github.com/eugenp/tutorials/assets/46810751/d6add599-339c-4706-a94e-aa152f2a73b3">
I use this codestyle scheme. 
made single line